apiTester/src/commonMain/kotlin/com/revenuecat/purchases/kmp/apitester/EntitlementInfoAPI.kt

```diff
@@ -47,7 +47,8 @@ private class EntitlementInfoAPI {
             val billingIssueDetectedAtInstant: Instant? = billingIssueDetectedAtInstant
             // FIXME re-enable in SDK-3530
             //  val ownershipType: OwnershipType = ownershipType
-            val verification: VerificationResult = verification
+            // FIXME re-enable in SDK-3530
```

Comment:
I saw we have had to do this before... I can't say I fully understand it, since the IDE seems to find the correct classes, but it's using different classes when trying to compile the api tests... In any case, I do believe this should be fixed once we move more classes to remove typedefs

Reply:
Yea I think this is the same issue as in the previous comment.